File: Applications/Python/Pointers/TwoSum.py
from typing import List

def twoSum(self, numbers: List[int], target: int) -> List[int]:
    left = 0
    right = len(numbers) - 1

    while left < right:
        sum = numbers[left] + numbers[right]

        if target == sum:
            return [left + 1, right + 1]
        elif sum < target:
            left += 1
        else:
            right -= 1

    return []

File: Applications/CSharp/Pointers/TwoSum.cs
int[] TwoSum(int[] numbers, int target)
{
    var left = 0;
    var right = numbers.Length - 1;

    while (left < right)
    {
        var sum = numbers[left] + numbers[right];
        if (sum == target)
        {
            return new int[] { left + 1, right + 1 };
        }
        else if (sum < target)
        {
            left++;
        }
        else
        {
            right--;
        }
    }

    return new int[0];
}

